usr/lib/common/aix/asprintf.h

```diff
@@ -0,0 +1,61 @@
+/*
+ * COPYRIGHT (c) International Business Machines Corp. 2024
+ *
+ * This program is provided under the terms of the Common Public License,
+ * version 1.0 (CPL-1.0). Any use, reproduction or distribution for this
+ * software constitutes recipient's acceptance of CPL-1.0 terms which can be
+ * found in the file LICENSE file or at
+ * https://opensource.org/licenses/cpl1.0.php
+ */
+#ifndef UTIL_LINUX_ASPRINTF_H
+#define UTIL_LINUX_ASPRINTF_H
+
+#include <stdlib.h>
+#include <stdio.h>
+#include <stdarg.h>
+
+#ifndef HAVE_VASPRINTF
+static inline
+int vasprintf(char **strp, const char *fmt, va_list ap)
+{
+	int str_size = -1, len;
```

Comment:
Can you please use the opencryptoki indention style (4 blanks) hin this entire file?